draft-ietf-tls-tls13.md

```diff
@@ -3532,6 +3571,25 @@ schedule with a black-box HKDF API, albeit at some loss of efficiency
 as some HKDF-Extract operations will be repeated.
 
 
+## Updating Traffic Keys
+
+Once the handshake is complete, it is possible for either side to
+update its sending traffic keys using the KeyUpdate handshake message
+{{key-update}}.  The next generation of traffic keys is computed by
+generating traffic_secret_N+1 from traffic_secret_N as described in
+this section then re-deriving the traffic keys as described in
+{{traffic-key-calculation}}.
+
+The next-generation traffic_secret is computed as:
+
+  traffic_secret_N+1 = HKDF-Expand-Label(traffic_secret_N,
+                                       "traffic_secret", "", L)
```

Comment:
The string probably shouldn't have the underscore, as the other secrets including the prior traffic secret N do not.